app/run.py

```diff
@@ -77,106 +86,80 @@
 
 def callback(pubsub_message):
 
-    try:
-        log_message = json.loads(pubsub_message.data)
-        log_id = log_message.get('insertId', 'unknown-id')
-
-        # Get the timestamp from the log message
-        log_time_str = log_message.get('timestamp')
-        if log_time_str:
-            log_timestamp = int(dateutil.parser.parse(log_time_str).timestamp())
-        else:
-            log_timestamp = int(time.time())
+    # We use the message ID in all logs we emit
+    message_id = pubsub_message.message_id
+    message_timestamp = int(pubsub_message.publish_time.timestamp())
 
+    try:
+        message_data = json.loads(pubsub_message.data)
     except (json.JSONDecodeError, AttributeError):
         # We can't parse the log message, nothing to do here
-        logger.debug('Failure loading json, discarding message')
+        logger.debug({'message_id': message_id, 'message': 'Failure loading json, discarding message'})
         pubsub_message.ack()
         return
 
-    logger.debug({'log_id': log_id, 'message': 'Received & decoded json message'})
-
-    # normal activity logs have logName in this form:
-    #  projects/<p>/logs/cloudaudit.googleapis.com%2Factivity
-    # data access logs have a logName field that looks like:
-    #  projects/<p>/logs/cloudaudit.googleapis.com%2Fdata_access
-    #
-    # try to only handle the normal activity logs
-    log_name_end = log_message.get('logName', '').split('/')[-1]
-    if log_name_end != 'cloudaudit.googleapis.com%2Factivity':
-        logger.debug({'log_id': log_id, 'message': 'Not an activity log, discarding'})
-        pubsub_message.ack()
-        return
+    logger.debug({'message_id': message_id, 'message': 'Received & decoded json message'})
```

Comment:
please split these long logger messages onto multiple lines

Reply:
I added a flake8 config to tox.ini and set it to 120chars for max line length. That's far more reasonable than the default 79 chars. I fixed all lines violating that rule.

Reply:
okay, that should do it